src/py/aspen/test_models/fixtures.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from ..database.models.usergroup import Group, User
+
+
+@pytest.fixture(scope="function")
+def group(session):
+    group = Group(name="groupname", email="groupemail", address="123 Main St")
+    session.add(group)
+    session.commit()
```

Comment:
why is that better than doing `session.commit()`?

Reply:
As a general practice, I just use `flush()` unless I actually want it persisted to the database.

Reply:
so when i do flush the tests break,  so i added the commit back in. i think since the test client is establishing a new session scope in the usergroup view that this needs to be committed?

Reply:
Ah! Yes, that is a great reason to do the commit.